v2/sourcedb-to-spanner/src/main/java/com/google/cloud/teleport/v2/source/reader/io/cassandra/iowrapper/CassandraDataSource.java

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright (C) 2024 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package com.google.cloud.teleport.v2.source.reader.io.cassandra.iowrapper;
+
+import com.google.auto.value.AutoValue;
+import com.google.common.collect.ImmutableList;
+import java.io.Serializable;
+import java.net.InetSocketAddress;
+import java.util.List;
+
+/**
+ * Encapsulates details of a Cassandra Cluster. Cassandra cluster logically maps to a Mysql or PG
```

Comment:
Cassandra cluster logically maps to a MySQL or PG instance.
is this a typo ?

Reply:
It shows an equivalelce in the comments for someone who's coming from a mySQL point of view. Though it's not one-to-one as in Cassandra also needs a list of INet addresses and not just a clusterName.

Actually I will remove this, in case it's not clear.

Reply:
Fixed. Removed the confusing part of the comparison. Thanks.